examples/pingone-neo-getting-started/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    pingone = {
+      source  = "pingidentity/pingone"
+      version = "~> 0.16.0"
```

Comment:
This will need to be a minimum of `0.17.1` because of SDK fixes

Reply:
fixed